R/chrome.R

```diff
@@ -95,10 +95,13 @@ chrome_print = function(
   ))
 
   debug_port = servr::random_port()
+  log_file = if (isTRUE(getOption('pagedown.keep.chrome.log'))) {
+    sprintf('chrome-stderr-%i.log', as.integer(Sys.time()))
```

Comment:
Does it look better and more human-readable if we use the actual time instead of converting it to integer? e.g., using a filename like `chrome-stderr-2021-08-13 21:23:15.log`

```suggestion
    sprintf('chrome-stderr-%s.log', Sys.time())
```